core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -70,10 +70,6 @@
  * respective unpooled versions that never leak, so that the {@link Subscriber} does not need to worry about
  * leaks.
  *
- * <p>If a {@link Subscriber} does not want a {@link StreamMessage} to make a copy of a {@link ByteBufHolder},
- * specify {@link SubscriptionOption#WITH_POOLED_OBJECTS} when you subscribe. Note that the {@link Subscriber}
- * is responsible for releasing the objects given with {@link Subscriber#onNext(Object)}.
```

Comment:
Removed this section because it's not valid anymore and it's documented well already elsewhere.